string uuid
float32 start_time
float32 previous_time
float32 current_time
float32 stop_time
